schema.sql

```diff
@@ -9,20 +9,21 @@ CREATE TABLE popularDishes (
     dish_name VARCHAR(40), 
     top_dish_photo VARCHAR(60),
     restaurant_name VARCHAR(40), 
-    photo_count INT, 
-    review_count INT, 
-    review_id INT, 
-    reviewer_name VARCHAR(40), 
     photo_id INT);
 
```

Comment:
Can you explain your schema?  If each popular dish can have multiple photos and reviews, I'd have tables for photos and reviews, where each photo/review is tied to the dish id.  I would not have a field in the dish table for photo id or review id because this could have multiple values.